src/client/gui/lib/main.dart

```diff
@@ -36,11 +36,13 @@ void main() async {
     return screen?.frame.size;
   });
 
-  final windowSize = (screenSize != null &&
-          screenSize.width >= 1600 &&
-          screenSize.height >= 900)
+  final windowSize = (screenSize != null)
+    ? (screenSize.width >= 1600 && screenSize.height >= 900)
       ? const Size(1400, 822) // For screens 1600x900 or larger
-      : const Size(750, 450); // Default window size
+      : (screenSize.width >= 1280 && screenSize.height >= 720)
+        ? const Size(1024, 576) // For screens 1280x720 or larger
+        : const Size(750, 450) // Default window size
+    : const Size(750, 450); // Default window size if screenSize is null
```

Comment:
If screenSize is null it means that the plugin cannot get the current screen resolution for some reason. Falling back to the minimum resolution is probably safest if this happens

Reply:
I can't think of any situations where serving a larger resolution when you don't know the size of the screen is safer than serving the minimum resolution

Reply:
Safety is perhaps not the best concept here, but when "the plugin cannot get the current screen resolution for some reason", the most reasonable expectation is still that the user either doesn't have graphics at all, or they have 720p or higher (since most screens today have that). OTOH, the smallest resolution is our least preferred. 

We can deal with this better later.